examples/Remarketing/UploadConversionEnhancement.php

```diff
@@ -241,7 +241,7 @@ public static function runExample(
         // Prints the status message if any partial failure error is returned.
         // Note: The details of each partial failure error are not printed here, you can refer to
         // the example HandlePartialFailure.php to learn more.
-        if (!is_null($response->getPartialFailureError())) {
```

Comment:
I could find 5 more occurrences of this in other code examples, could you change them as well in this PR?